sylt-parser/src/statement.rs

```diff
@@ -217,18 +236,18 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
                 parent.join(if path == "/" {
                     format!("exports.sy")
                 } else if path.ends_with("/") {
-                    format!("{}/exports.sy", bare_name)
+                    format!("{}/exports.sy", name)
                 } else {
-                    format!("{}.sy", bare_name)
+                    format!("{}.sy", name)
                 })
             };
-            let (ctx, ident) = match &ctx.tokens[ctx.curr..] {
+            let (ctx, alias) = match &ctx.tokens_forward::<2>() {
                 [T::As, T::Identifier(alias), ..] => (
```

Comment:
```suggestion
                [T::As, T::Identifier(alias)] => (
```

Reply:
Disagree. `..` means that we can change the lookahead to 3 (or something else) without needing to change every branch.